### PR TITLE
Update bulletin board documentation

### PR DESCRIPTION
**Pull request by slickepinne**
This PR updates the documentation of the Bulletin Board. It follows the structure of the Decidim documentation. I didn't change the  README.md files of the gems yet.

### CODE REVIEW

File: docs/modules/develop/pages/environment.adoc [hunk 1]
```diff
@@ -1,5 +1,60 @@
 = Development environment
 
-To start working with the Bulletin Board you will need to install all the dependencies for the server. These are explained in the https://github.com/decidim/decidim-bulletin_board/blob/main/README.md[README file of the web application]. This server will include test data that can be used by Decidim or the elections sandbox to test the different parts of the system.
+**Important: The Bulletin Board depends on the existence of a Decidim installation.**
 
-Also, this application includes Visual Studio Code Dev Containers to simplify the environment setup.
+To start working with the Bulletin Board you will need to install all the dependencies for the server.
```
Comment (leio10) [inline docs/modules/develop/pages/environment.adoc:5]: @beagleknight is working on [a PR to add make for installing and updating the different parts of the project](https://github.com/decidim/decidim-bulletin-board/pull/155), maybe we should add that here?

File: docs/modules/develop/pages/guide_commands.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+= Commands
+
+Here are some useful commands and tools for working in this project:
```
Comment (leio10) [inline docs/modules/develop/pages/guide_commands.adoc:3]: Same as before here

File: docs/modules/install/pages/index.adoc [hunk 3]
```diff
@@ -5,50 +5,44 @@
 
 Decidim Bulletin Board is an independent application designed to provide services to Decidim instances to allow them to celebrate secure end-to-end auditable votings. But it can't be used as a standalone application, since it doesn't offer a user interface for administrators to define elections nor for voters to cast their votes.
 
-If you want to start your own installation of Decidim Bulletin Board, you need to clone this repo. Keep reading to find out how to install Decidim Bulletin Board.
+If you want to start your own installation of the Decidim Bulletin Board, you need to clone this repo. Keep reading to find out how to install and setup a Decidim Bulletin Board.
 
-== Clone the Decidim Bulletin Board repository
+== Installing
```
Comment (leio10) [inline docs/modules/install/pages/index.adoc:10]: Same as before here.

**Review by leio10 — APPROVED** [review]
Awesome and so needed work!! 🤩 📝

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+= Decidim Bulletin Board
+
+Decidim Bulletin Board is an independent application, written in Ruby on Rails, JavaScript and Python, designed to provide services to https://decidim.org[Decidim] instances to allow them to celebrate secure end-to-end auditable votings. Although it can't be used as a standalone application, since it doesn't offer a user interface for administrators to define elections nor for voters to cast their votes, you can see how it works in the https://decidim-bulletin-board-staging.herokuapp.com/sandbox/elections[sandbox].
```
[the remark this answers is not in the document]
Comment (slickepinne) [reply README.adoc:3]: That's true. I'll update it! 😊

File: docs/modules/develop/pages/environment.adoc (hunk 1 above)
Comment (slickepinne) [reply docs/modules/develop/pages/environment.adoc:5]: As soon as it's ready, I'll update the documentation again. This was mainly for restructuring and adding some missing parts, like **Adding an authority**.

**Review by orlera — APPROVED** [review]
Love it! ❤️

**Comment by agustibr** [comment]
Thanks @slickepinne for taking care of the docs 😀 ✨

**Pull request merged**